Library/Formula/knot.rb

```diff
@@ -47,41 +45,26 @@ def install
     (var/"knot").mkpath
   end
 
-  def knot_conf; <<-EOS.undent
-    system {
-      identity on;
-      version on;
-      rundir "#{var}/knot";
-    }
-    interfaces {
-      all_ipv4 {
-        address 0.0.0.0;
-        port 53;
-      }
-      all_ipv6 {
-        address [::];
-        port 53;
-      }
-    }
-    control {
-      listen-on "knot.sock";
-    }
-    zones {
-      storage "#{var}/knot";
-    #  example.com {
-    #    file "#{var}/knot/example.com.zone";
-    #  }
-    }
-    log {
-      syslog {
-        any error;
-        zone warning, notice;
-        server info;
-      }
-      stderr {
-        any error, warning;
-      }
-    }
+  def knot_conf; <<-EOS
+server:
+  rundir: "/usr/local/var/knot"
```

Comment:
Should use `#{var}` rather than `/usr/local/var`. Can't assume Homebrew lives in `/usr/local`